src/cli/backup/backup.go

```diff
@@ -187,8 +190,15 @@ func genericCreateCommand(
 
 		bo, err := r.NewBackupWithLookup(ictx, discSel, ins)
 		if err != nil {
-			errs = append(errs, clues.Wrap(err, owner).WithClues(ictx))
-			Errf(ictx, "%v\n", err)
+			cerr := clues.Wrap(err, owner).WithClues(ictx)
+			errs = append(errs, cerr)
+
+			clj, jmerr := json.Marshal(cerr.Core().Values)
+			if jmerr != nil {
+				clj = []byte("Unable to marshal error metadata")
+			}
+
+			Errf(ictx, color.Red("ERROR: Unable to complete backup")+" \nMessage: %v\nMetadata: %s\n", err, clj)
```

Comment:
Good catch. I've kept it to `Error` as that is what cobra also does. As for the reason I moved the `Error` string from `Errf`, for one it was inconsistent with `Err`, also, I wanted to just color the header part of the error and not the entire error.

As a middle ground I have introduced two new functions `SimpleError` and `OperationError` instead of `Err` and `Errf`.